src/res/flame.html

```diff
@@ -51,24 +230,83 @@ <h1>/*title:*/</h1>
 	if (devicePixelRatio) c.scale(devicePixelRatio, devicePixelRatio);
 	c.font = document.body.style.font;
 
+	function positionTooltip(e) {
```

Comment:
Does it even need to float? Static position will be more than enough. It should make code simpler.

Reply:
"Need" is a strong word for a UI choice. Made it static anyway.